services/stream_service.py

```diff
@@ -9,6 +9,8 @@ class CreateStream(webapp2.RequestHandler):
     def post(self):
 
         json_string = self.request.body
+        logging.info("json-string")
```

Comment:
Is it still needed to log the json string?

Reply:
Nope. That's my bad. Will remove.